kivymd/uix/button.py

```diff
@@ -1793,6 +1793,22 @@ def callback(self, instance):
     and defaults to `False`.
     """
 
+    auto_dismiss = BooleanProperty(False)
+    """
+    Whether to dismiss stack when clicked out of focus of widget.
+
+    :attr:`auto_dismiss` is a :class:`~kivy.properties.BooleanProperty`
+    and defaults to `False`.
+    """
+
+    dismiss_on_option_touch = BooleanProperty(False)
+    """
+    Whether to dismiss stack when clicked on one of the options.
+
+    :attr:`dismiss_on_option_touch` is a :class:`~kivy.properties.BooleanProperty`
+    and defaults to `False`.
+    """
+
```

Comment:
It seems to me that the `dismiss_on_option_touch` parameter is superfluous. It is enough to specify the `auto_dismiss` parameter.

Reply:
ok i'll remove redundant feature alone